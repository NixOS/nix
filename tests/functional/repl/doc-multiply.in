:l doc-comments.nix
:doc multiply
