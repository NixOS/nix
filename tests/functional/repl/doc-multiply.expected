Nix <nix version>
Type :? for help.
Added <number omitted> variables.

Function multiply
    â€¦ defined at
    /path/to/tests/functional/repl/doc-comments.nix:10:14

    Perform arithmetic multiplication. It's kind of like
    repeated addition, very neat.

      | multiply 2 3
      | => 6


